http-handler-with-stats.go

```diff
@@ -25,6 +25,9 @@ func HTTPHandlerWithStats(routeName string, router http.Handler, logger logger,
 
 func logResult(routeName string, metrics httpsnoop.Metrics, statsd StatsD, logger logger, url string) {
 	statsd.Histogram("web.response_time", float64(metrics.Duration.Nanoseconds())/1000000, "route:"+routeName, "response:"+strconv.Itoa(metrics.Code))
+	responseCodeKey := fmt.Sprintf("web.response_code.%s", strconv.Itoa(metrics.Code))
```

Comment:
since you're doing Sprintf anyway, couldn't you just do fmt.Sprintf("web.response_code.%d", metrics.Code) ?